src/routes/requestRoute.js

```diff
@@ -12,6 +12,14 @@ const requestRoutes = app => {
     Authorization.checkToken,
     Validate.validateUserInput,
     RequestController.bookAReturnTrip);
+
+  app.get('/api/v1/requests', Authorization.checkToken,
```

Comment:
Let all parameters in the `app.get` method be on a single line.

Reply:
Okay

Reply:
Eslint says it's too long